devices/aubess.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable linebreak-style */
+const exposes = require('../lib/exposes');
+const fz = require('../converters/fromZigbee');
+const tz = require('../converters/toZigbee');
+const ea = exposes.access;
+
```

Comment:
Please move all the additions to `toZigbee.js` and `fromZigbee.js` to here (`fzLocal`/`tzLocal` like https://github.com/Koenkk/zigbee2mqtt.io/blob/ce7d93ca3cf736b9a30f75fa73e7b4002d9bf8eb/docs/externalConvertersExample/freepad_ext.js#L20)

Reply:
I plan (in nearest future) to support a MOES device which use same protocol to handle IR remote control (look at the source issue).  That's why I move this part of the code to `toZigbee.js` and `fromZigbee.js`.
I agree that jumped the gun, and I'll move it to `device/aubess.js`.

But could you explain shall I move that code back to `toZigbee.js` and `fromZigbee.js` when I add a MOES device, or I should just copy it to `moes.js` file?

Reply:
I see, maybe better to move to `lib/zosung.js` then (btw where does the name zosung come from?)

Reply:
I have a small investigation and figure out that the devices are likely manufactured by Shenzhen Zhuoxing Smart Technology Co., Ltd. (Zosung, [device page](http://www.zosung.cn/en/productshow.asp?Id=263)) for different vendors like AUBESS and so on.

You suggestion look reasonable (less conflicts and so on). I'll move everything into `lib/zosung.js` and make a commit after checking I break nothing.

Reply:
I moved everything to `lib/zosung.js`.
All the changes in `toZigbee.js` and `fromZigbee.js` has been reverted.